src/Filters/ExactMatch.php

```diff
@@ -100,9 +115,42 @@ abstract protected function getBlacklist();
     /**
      * Get a list of file paths to include.
      *
+     * @deprecated 3.9.0 Overload the `getAllowedFiles()` method instead.
```

Comment:
100% pedantic detail, "overwrite" (or "reimplement") instead of "overload" ?

Reply:
Fixed to "Implement".